tools/automerge_check.yml

```diff
@@ -0,0 +1,7 @@
+script: 
```

Comment:
Renaming to something other than [a protected name for automated package finding / building](https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#setuptools.discovery.FlatLayoutPackageFinder.DEFAULT_EXCLUDE) (e.g. `docs`, `scripts`, `tools`) means two package-level directories. This can be fixed with the appropriate specs in the `pyproject.toml` setuptools section but it does add some overhead.